source/Octopus.Tentacle/Kubernetes/ScriptPodSinceTimeStore.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Concurrent;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Kubernetes
+{
+    public interface IScriptPodSinceTimeStore
+    {
+        DateTimeOffset? GetSinceTime(ScriptTicket scriptTicket);
+        void UpdateSinceTime(ScriptTicket scriptTicket, DateTimeOffset nextSinceTime);
+        void Delete(ScriptTicket scriptTicket);
+    }
+
+    public class ScriptPodSinceTimeStore : IScriptPodSinceTimeStore
```

Comment:
I decided to not put this on the tracked pods, this means the monitoring is not coupled to the log reading